common/fe-ste-models.go

```diff
@@ -934,6 +938,11 @@ func (m Metadata) ToAzFileMetadata() azfile.Metadata {
 	return azfile.Metadata(m)
 }
 
+// ToAzBlobTagsMap converts BlobTagsMap to azblob's BlobTagsMap
+func (bt BlobTags) ToAzBlobTagsMap() azblob.BlobTagsMap {
+	return azblob.BlobTagsMap(bt)
+}
```

Comment:
please pull the BlobTags code together for readability.

Reply:
Done